src/Raven.Server/RavenServer.cs

```diff
@@ -2947,6 +2948,28 @@ internal class DebugPackageTestingStuff
             {
                 internal string[] RoutesToSkip = new string[] { };
             }
+            internal Action<int> ReservePortSocket;
```

Comment:
Lets have meaningful name:
```suggestion
            internal Action<int> OnSimulateRunningServerFinally;
```

Reply:
done